gateway/testutil.go

```diff
@@ -1030,6 +1030,14 @@ func (s *Test) newGateway(genConf func(globalConf *config.Config)) *Gateway {
 	gw := NewGateway(gwConfig, s.ctx)
 	gw.setTestMode(true)
 
+	// This overrides the default dialer for redis to always use 127.0.0.1 in tests;
```

Comment:
I wonder if using 127.0.0.1 works inside github actions?

Reply:
@buger passing tests say yes, this would apply